index.html 

```diff
@@ -27,7 +27,7 @@
 		<button>More</button>
 	</div>
 	<footer>
-		<a href=#copyright">copyright 2022</a>
+		<a href=#copyright">copyright 2023</a>
```

Comment:
This should actually be 2024

Reply:
Are you sure? The website is active as of 2023. @dlamdevelops

Reply:
Yes, that's correct. @blam-codes